client/network-manager.go

```diff
@@ -214,7 +214,8 @@ loop:
 				}
 			}
 			// Update config
-			data.MergePoints(nodePoints.ID, nodePoints.Points, &c.config)
+			err := data.MergePoints(nodePoints.ID, nodePoints.Points, &c.config)
```

Comment:
Should check for error using `if(err != nil)` before printing

Reply:
thanks, fixed.